usaspending_api/api_docs/api_documentation/budget_authority/agencies.md

```diff
@@ -0,0 +1,126 @@
+
+## Budget Athority Agencies
+**Route:** `/api/v2/budget_authority/agencies/(?P<cgac>\w+)/$'`
```

Comment:
Change the route to be user-friendly: `/api/v2/budget_authority/agencies/:cgac`

Reply:
This route is not used by the frontend; remove from documentation and disable the URL.

Reply:
gone